### PR TITLE
(5.x.x) Re-enable restore progress bar 

### PR DESCRIPTION
**Pull request by dizzzz**
Port of #2439

### CODE REVIEW

File: src/org/exist/backup/FileSystemBackupDescriptor.java [hunk 1]
```diff
@@ -33,9 +35,13 @@
 import java.nio.file.Path;
 import java.nio.file.Paths;
 import java.util.Properties;
+import java.util.stream.Stream;
 
 
 public class FileSystemBackupDescriptor extends AbstractBackupDescriptor {
+
+    private final static Logger LOG = LogManager.getLogger();
```
Comment (adamretter) [inline src/org/exist/backup/FileSystemBackupDescriptor.java]: Don't we need a class here for the logger? e.g. `getLogger(AbstractBackupDescriptor.class)`?

File: src/org/exist/backup/ZipArchiveBackupDescriptor.java [hunk 2]
```diff
@@ -42,6 +44,8 @@
 
 public class ZipArchiveBackupDescriptor extends AbstractBackupDescriptor {
 
+    private final static Logger LOG = LogManager.getLogger();
```
Comment (adamretter) [inline src/org/exist/backup/ZipArchiveBackupDescriptor.java]: Don't we need a class here for the logger? e.g. `getLogger(ZipArchiveBackupDescriptor.class)`?

Comment (dizzzz) [reply src/org/exist/backup/ZipArchiveBackupDescriptor.java]: AFAIK this is not needed (anymore). in the era of log4j 1.2.x is was indeed required for some time.

see https://logging.apache.org/log4j/2.x/log4j-api/apidocs/org/apache/logging/log4j/LogManager.html#getLogger()

File: src/org/exist/backup/FileSystemBackupDescriptor.java (hunk 1 above)
Comment (dizzzz) [reply src/org/exist/backup/FileSystemBackupDescriptor.java]: AFAIK this is not needed (anymore). in the era of log4j 1.2.x is was indeed required for some time.

see https://logging.apache.org/log4j/2.x/log4j-api/apidocs/org/apache/logging/log4j/LogManager.html#getLogger()

**Pull request merged**